src/Modal.js

```diff
@@ -30,13 +92,20 @@ const Modal = React.createClass({
       backdrop: true,
       keyboard: true,
       animation: true,
-      closeButton: true
+      closeButton: true,
+      enforceFocus: true
     };
   },
 
+  getInitialState(){
+    return { };
+  },
+
   render() {
-    let modalStyle = {display: 'block'};
+    let state = this.state;
+    let modalStyle = { ...state.dialogStyles, display: 'block'};
```

Comment:
`{ ...state.dialogStyles, display: 'block'}`
just for symmetry
`{ ...state.dialogStyles, display: 'block' }`
or
`{...state.dialogStyles, display: 'block'}`